src/components/Button.tsx

```diff
@@ -4,41 +4,42 @@ import styled from 'styled-components';
   FOR PRIMARY BUTTON USAGE:
 
     Props: 
-      primaryColor (Required) - This color will determine the default background color of the button
-      secondaryColor (Required) - This color will determine the border color and background color on hover
+      $primaryColor (Required) - This color will determine the default background color of the button
+      $secondaryColor (Required) - This color will determine the border color and background color on hover
    
     Example:
-      <Button primaryColor={COLORS.primaryBlue} secondaryColor={COLORS.primaryBlueClicked} onClick={...}>
+      <Button $primaryColor={COLORS.blueMid} $secondaryColor={COLORS.blueDark} onClick={...}>
         [Button text here]
       </Button>
 
   FOR SECONDARY BUTTON USAGE:
 
     Props:
-      primaryColor (Omit) - IMPORTANT: DO NOT PASS IN A primaryColor PROP!!! The background color will be white by default
-      secondaryColor (Required) - This color will determine the border color and background color on hover
+      $primaryColor (Omit) - IMPORTANT: DO NOT PASS IN A primaryColor PROP!!! The background color will be white by default
+      $secondaryColor (Required) - This color will determine the border color and background color on hover
 
     Example: 
-      <Button secondaryColor={COLORS.primaryBlue} onClick={...}>
+      <Button $secondaryColor={COLORS.blueMid} onClick={...}>
         [Button text here]
       </Button>
  */
 const Button = styled.button<{
-  primaryColor?: string;
-  secondaryColor: string;
+  $primaryColor?: string;
+  $secondaryColor: string;
 }>`
   appearance: none;
-  color: ${props => (props.primaryColor ? 'white' : 'black')};
-  background: ${props => (props.primaryColor ? props.primaryColor : 'white')};
+  color: ${props => (props.$primaryColor ? 'white' : 'black')};
+  background: ${props => (props.$primaryColor ? props.$primaryColor : 'white')};
   align-self: flex-end;
   padding: 0.625rem 1.25rem;
   border-radius: 0.313rem; // 5px
   border: 2px solid
-    ${props => (props.primaryColor ? props.primaryColor : props.secondaryColor)};
+    ${props =>
+      props.$primaryColor ? props.$primaryColor : props.$secondaryColor};
   &:hover {
```

Comment:
this should also be the case for `&:focus`

Reply:
Note [worksesh]: moving forward with keeping the outline for the `Button` component since adding styles to `&:focus` state results in weird behavior. Specifically, after clicking the button, it will remain in focus state. When the form has errors, it kinda looks like the button is stuck. Discussed with @davidqing6432 that we should not add a custom `outline-color` and allow for the outline color to default to whatever the user's outline color is on their device.